app/Http/Controllers/Web/ResetPasswordController.php

```diff
@@ -136,6 +136,10 @@ protected function guard()
      */
     public function redirectPath()
     {
+        if (Str::contains(request()->type, 'activate-account')) 
+        {
+        return 'profile/about';
+        }
```

Comment:
I think this is what it wants:
```suggestion
        if (Str::contains(request()->type, 'activate-account')) {
            return 'profile/about';
        }

```